### DISCUSSION TITLE
Tell me your hobby!

### DISCUSSION THREAD

**Discussion by satorukano** [NAIST-SE/AttractingContributors#1123] [2024 Attraction Contributions]
https://github.com/satorukano/AttractingContributors_hobby

Please tell me your hobby!

**Comment by mustafizur-r** [comment]
Hello [@satorukano](https://github.com/satorukano) , I recently made a contribution to your project , I have added “Cricket" . 
could you please contribute to my as well ? Following is the link to [my repository](https://github.com/MustafizurRU/AttractingContributors-My_Favorite_Sci-Fi_Books) . Thanks in advance.

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
